### PR TITLE
Comments related to Alcotest_ext

### PR DESCRIPTION
**Pull request by mjambon**

### CODE REVIEW

File: test [hunk 1]
```diff
@@ -1,12 +1,7 @@
 #! /usr/bin/env bash
 #
-# Run the semgrep-core test program from the correct folder.
-# This is meant for manual use, when we want to select which tests to run.
+# This calls the test program generated by alcotest_ext.
+# We're not using a symlink because dune rejects dangling symlinks.
```
Comment (aryx) [inline test]: really? but bin/ is a dangling symlink now and dune is fine with it.

**Review by aryx — APPROVED** [review]

File: test (hunk 1 above)
Comment (mjambon) [reply test]: oh, that's great. It used to be a problem (possibly https://github.com/ocaml/dune/issues/3819)

**Comment by mjambon** [comment]
```
check for broken symlinks................................................Failed
```
Why is this failing in CI? It's working for me locally:
```
$ pre-commit run check-symlinks -a
check for broken symlinks................................................Passed
```
Is it using the pre-commit config from the main branch? I'm going to assume this and force-merge.

**Comment by aryx** [comment]
it's working locally because you've probably called make, so your _build/ exists.
In CI we start from a fresh clone without any _build, so then the test and meta-test are broken symlinks.
Anyway it's fixed in 1.54.1 by @amchiclet by whitelisting those links in .pre-commit-config.yaml

**Comment by aryx** [comment]
(just like I did for the bin toplevel symlink a few weeks ago).

**Pull request merged**